Content.Shared/EmagFixer/Components/EmagFixerComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Shared.EmagFixer.Components
+{
+    [RegisterComponent]
+    public sealed class EmagFixerComponent : Component
+    {
+        [DataField("maxCharges"), ViewVariables(VVAccess.ReadWrite)]
+        public int MaxCharges = 4;
+
+        [DataField("charges"), ViewVariables(VVAccess.ReadWrite)]
+        public int Charges = 4;
+    }
+}
```

Comment:
IMO all of these should be in the same namespace as regular emag stuff. same goes with the serverside emagfixer system.